cc_core.c

```diff
@@ -128,13 +128,45 @@ char* register_from_string(int reg)
 	exit(EXIT_FAILURE);
 }
 
+int hex2char(int c)
+{
+	if((c >= 0) && (c <= 9)) return (c + 48);
+	else if((c >= 10) && (c <= 15)) return (c + 55);
+	else return -1;
+}
+
+char* number_to_hex(int a, int bytes)
```

Comment:
can we not use int2str from M2libc/bootstrappable.c?

Reply:
I just tested and sadly not. `number_to_hex` pads to `bytes` while `int2str` doesn't.